src/searcher.cpp

```diff
@@ -756,13 +756,12 @@ Clause* Searcher::analyze_conflict(
     stats.litsRedNonMin += learnt_clause.size();
     minimize_learnt_clause<update_bogoprops>();
     stats.litsRedFinal += learnt_clause.size();
+    glue = calc_glue(learnt_clause);
     if (learnt_clause.size() <= conf.max_size_more_minim) {
-        glue = calc_glue(learnt_clause);
         if (glue <= conf.max_glue_more_minim) {
             minimize_using_permdiff();
```

Comment:
After this line, you need to re-calculate the glue. It might have changed, since minimization can reduce the size of the clause, thereby changing the glue in it.